### DISCUSSION TITLE
Bibtex Citation

### DISCUSSION THREAD

**Discussion by noamsgl** [Lightning-AI/pytorch-lightning#13284] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hello. I love PL!

is there a bibtex version of this citation out there?

Thanks

https://github.com/PyTorchLightning/pytorch-lightning/blob/bfa8b7be2d99b980afa62f5cb0433326bcfd2ef0/CITATION.cff#L1

**Comment by akihironitta** [comment, marked as answer]
@noamsgl You can copy it from "Cite this repository" in the side bar of the top page.

![Screen Shot 2022-06-15 at 22 42 09](https://user-images.githubusercontent.com/20610905/173842305-2f939fa1-2bf6-4404-8890-fd693b4b3be2.png)
